app/assets/stylesheets/responsive/custom.scss

```diff
@@ -1659,6 +1659,10 @@ p.locale-list-trigger {
   ul {
     list-style: none outside none;
     padding: 0;
+
```

Comment:
Ordinarily I'd suggest a separate pull request for completely orthogonal commits, but since it's here then let's just stick with it 👍